### PR TITLE
Update versions and changelog from latest releases.

### LINKED ISSUE #27003: Master branch still says Version: 4.3.0-dev
The master branch still says `Version: 4.3.0-dev` in the woocommerce.php plugin header. Should this be either 4.3.0 or dev for the next version?

### PR DESCRIPTION
**Pull request by peterfabian**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Updating versions and changelog after 4.3 release.

Closes #27003

### CODE REVIEW

File: includes/class-woocommerce.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ final class WooCommerce {
 	 *
 	 * @var string
 	 */
-	public $db_version = '430';
+	public $db_version = '440';
```
Comment (vedanshujain) [inline includes/class-woocommerce.php:32]: you don't need to update DB version every release but only if there is a change in any of the base table.

**Review by vedanshujain — COMMENTED** [review]
One comment needs to be addressed, other than that LGTM

**Review by vedanshujain — APPROVED** [review]
LGTM

**Review by claudiosanches — APPROVED** [review]
LGTM!

**Pull request merged**